IntelliTectAnalyzer/IntelliTectAnalyzer.Integration.Tests/MSBuildLocatorInitializer.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace IntelliTectAnalyzer.Integration.Tests
+{
+    internal static class MSBuildLocatorInitializer
+    {
+        static MSBuildLocatorInitializer()
+        {
+            VisualStudioInstance =
+                Microsoft.Build.Locator.MSBuildLocator.RegisterDefaults();
```

Comment:
Is this the lib equivalent of vswhere.exe? if so, cool!

Reply:
Not really. This is the setup the needed stuff for the `MSBuildWorkspace`. So it is less about just locating VS and more about getting the needed assemblies loaded.